3_Repositories.Tests/HighlightRepositoryTest.cs

```diff
@@ -0,0 +1,171 @@
+using Models;
+using NUnit.Framework;
+using Repositories.Tests.Base;
+using Repositories.Tests.DataSources;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Repositories.Tests
+{
+    [TestFixture]
+    public class HighlightRepositoryTest : RepositoryTest<Highlight, HighlightRepository>
+    {
+        protected new IHighlightRepository Repository => (IHighlightRepository) base.Repository;
+
+        /// <summary>
+        /// Test if highlights are retrieved correctly
+        /// </summary>
+        /// <param name="highlights"></param>
+        /// <returns></returns>
```

Comment:
Unnecessary tags <param> and <returns>, since they are empty. Same for every other test comment.

Reply:
These are required by the coding guidelines.

Reply:
These tags are indeed required, but they must also be filled in.

Reply:
We should discuss what to do with this with the backend team.